tests/unit/test_types.py

```diff
@@ -208,7 +208,16 @@ def test_async_api_spec_from_and_to_dict(faker: Faker):
                 },
             }
         },
-        "servers": {"development": {"url": "localhost", "protocol": "ws"}},
+        "servers": {
+            "development": {
+                "url": "localhost",
+                "protocol": "ws",
+                "security": [{"test": []}],
+            }
+        },
+        "components": {
+            "securitySchemes": {"test": {"type": "http", "scheme": "basic"}}
```

Comment:
Would be good to add complete payloads of all the possible security schemes here so that we make sure that the forge functions do not break.